zerver/middleware.py

```diff
@@ -453,7 +452,7 @@ def csrf_failure(request: HttpRequest, reason: str = "") -> HttpResponse:
     if RequestNotes.get_notes(request).error_format == "JSON":
         return json_response_from_error(CsrfFailureError(reason))
     else:
-        return html_csrf_failure(request, reason)
+        return render(request, "404.html", context={"csrf_failure": reason}, status=403)
```

Comment:
What value is in the `reason` parameter? Should we be trying to display that?

Reply:
I have replied in the [topic](https://chat.zulip.org/#narrow/stream/101-design/topic/CSRF.20error.20page) so that the whole discussion is easy to follow up. I have also changed the context from `context={"csrf_failure": reason}` to `context={"csrf_failure": True}` for the time being.